src/routers/UserRouter.ts

```diff
@@ -88,6 +92,8 @@ const getProfileRouter = (basePath: string = '/', routePath: string = '/users/pr
       query('include').optional().isString().trim(),
       body('firstName').optional().isString().trim().notEmpty(),
       body('lastName').optional().isString().trim().notEmpty(),
+      body('country').optional().isString().trim().notEmpty(),
+      body('institution').optional().isString().trim(),
```

Comment:
Can we add here the same notEmpty() validator, since it's marked at optional, it shouldn't behave any different than country.

Reply:
We probably have to also check if the passed in Country is in the available set.

An example can be found here.
```
query('type').trim().isIn(['counter', 'shortid']),
```
https://github.com/natgeosociety/marapp-services/blob/EP-3682-add-new-profile-meta-data-fields/src/routers/LayerRouter.ts#L230